src/main/java/TaskManager.java

```diff
@@ -0,0 +1,124 @@
+/*
+import java.util.ArrayList;
+import java.util.List;
+*/
+
+/**
+ * Enum showing user actions in the Task Manager.
+ */
+/*
+public enum UserInput {
+    ADD_TODO,
+    ADD_DEADLINE,
+    ADD_EVENT,
+    DELETE_TASK,
+    LIST,
+    MARK,
+    UNMARK,
+    BYE;
+}
+*/
+
+/**
+ * This class manages the tasks and processes user actions.
+ */
+/*
+public class TaskManager {
+    private static final List<Task> THINGS_TO_DO = new ArrayList<>();
```

Comment:
im not sure if this variable should be declared as final and written in caps, since it is modified in the later part.